### PR TITLE
[PORT] Ecto Sniffer yells over Science Comms now

### PR DESCRIPTION
**Pull request by Bobbanz1**
## About The Pull Request
Ports the following PR from Upstream: 
* https://github.com/BeeStation/BeeStation-Hornet/pull/8229

That PR changes the Ecto Sniffer, making it able to send a message to science comms about ghosts demanding positronic brains rather than screaming the alert into an empty room with nobody around to hear it. 

The PR also makes it take 60 Seconds between individual activation rather than 30 Seconds.

## Why It's Good For The Game
Lowers the amount of materials wasted creating positronic brains that will just sit around doing nothing for hours, by making the Ecto Sniffer scream over Sci-comms when a ghost is demanding a cyborg body.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://user-images.githubusercontent.com/59128051/227172581-847e5a37-fbfe-4fa5-8ca7-fcf607e6573b.mp4

</details>

## Changelog
:cl:WhereAmO
tweak: Ecto Sniffer now sends a message over Science Comms when a Ghost uses it.
/:cl:

### CODE REVIEW

File: code/game/machinery/ecto_sniffer.dm [hunk 1]
```diff
@@ -34,11 +42,15 @@
 /obj/machinery/ecto_sniffer/proc/activate(mob/activator)
 	flick("ecto_sniffer_flick", src)
 	playsound(loc, 'sound/machines/ectoscope_beep.ogg', 25)
-	visible_message("<span class='notice'>[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
+	//NSV13 - Ecto Sniffer Radio Yelling - Start
+	var/msg = "[src] beeps, detecting ectoplasm! There may be additional positronic brain matrices available!"
+	radio.talk_into(src, msg, RADIO_CHANNEL_SCIENCE)
+	//NSV13 - Ecto Sniffer Radio Yelling - Stop
+	visible_message("<span class='notice'></span>")
```
Comment (covertcorvid) [inline code/game/machinery/ecto_sniffer.dm:49]: Is this an empty visible message?

Comment (Bobbanz1) [reply code/game/machinery/ecto_sniffer.dm:49]: Yes, it appears that I forgot to remove it

Comment (Bokkiewokkie) [reply code/game/machinery/ecto_sniffer.dm]: I like the idea that it just posts an empty line in everyone's chat bars

**Review by Bokkiewokkie — APPROVED** [review]
I kind of agree with Powerfulbacon's original opinion that the cooldown should be on the device itself. But you can always just turn the thing off if it's annoying.

**Comment by Bobbanz1** [comment]
> I kind of agree with Powerfulbacon's original opinion that the cooldown should be on the device itself. But you can always just turn the thing off if it's annoying.

Could do that, but I’d have to rework the entire file probably

**Pull request merged**